storage/nfs-utils/pkg.yaml

```diff
@@ -0,0 +1,43 @@
+name: nfs-utils
+variant: alpine 
+shell: /bin/bash
+install:
+  - bash
+  - build-base
+  - linux-headers
+  - libevent-dev
+  - libtirpc-dev
+  - libxml2-dev
+  - rpcgen
+  - util-linux-dev
+steps:
+  - sources:
+      - url: https://src.fedoraproject.org/lookaside/extras/nfs-utils/nfs-utils-{{ .NFS_UTILS_VERSION | replace "-" "." }}.tar.xz/sha512/ed8ac3a79f347a1dc91cba060f87f687eefc204430648099ca867114295dd8dd8e745bd809807d5bc79dfb1ad86eaf2a6f4506ec7889c54b231d8f65dcb59bd4/nfs-utils-{{ .NFS_UTILS_VERSION | replace "-" "." }}.tar.xz
+        destination: nfs-utils.tar.xz
+        sha256: {{ .NFS_UTILS_SHA256 }}
+        sha512: {{ .NFS_UTILS_SHA512 }}
+    prepare:
+      - |
+        tar -xf nfs-utils.tar.xz --strip-components=1
+
+        export PKG_CONFIG_PATH=/usr/lib/pkgconfig
+
+        ./configure \
```

Comment:
can the tool be compiled to look for config file under `/usr/local/etc`? Then it makes it easier to add that via an extensionserviceconfig